src/util/filter.js

```diff
@@ -99,6 +99,7 @@ export function matchesFilter(filter: IgnoreFilter, basename: string, loc: strin
   }
   return filter.regex.test(loc) ||
          filter.regex.test(`/${loc}`) ||
+         filter.regex.test(`\\${loc}`) ||
```

Comment:
The problem I see with keeping both `/` and `\\` is that the backslash has a meaning in unix systems, so false-positives could occur.